### PR TITLE
Log the master IPS when there is a login in error

### PR DESCRIPTION
**Pull request by mFelgate**
### Desired Outcome

Log the requests IP address when there is a login error

### Implemented Changes

Add a function to find and print he ip address taken from the host name.

- _What's changed? Why were these changes made?_
- _How should the reviewer approach this PR, especially if manual tests are required?_
- _Are there relevant screenshots you can add to the PR description?_

### Connected Issue/Story

CyberArk internal issue link: [ONYX-14476](https://ca-il-jira.il.cyber-ark.com:8443/browse/ONYX-14476)

### Definition of Done
Improved error message in authenticator client when request goes to an unhealthy master (502 error code doesn't reveal what IP address the name resolved to). Log the node's IP address when the connection fails.

#### Changelog

- [ ] The CHANGELOG has been updated, or
- [x] This PR does not include user-facing changes and doesn't require a
  CHANGELOG update

#### Test coverage

- [ ] This PR includes new unit and integration tests to go with the code
  changes, or
- [x] The changes in this PR do not require tests

#### Documentation

- [ ] Docs (e.g. `README`s) were updated in this PR
- [ ] A follow-up issue to update official docs has been filed here: [insert issue ID]()
- [x] This PR does not require updating any documentation

#### Behavior

- [ ] This PR changes product behavior and has been reviewed by a PO, or
- [ ] These changes are part of a larger initiative that will be reviewed later, or
- [x] No behavior was changed with this PR

#### Security

- [ ] Security architect has reviewed the changes in this PR,
- [ ] These changes are part of a larger initiative with a separate security review, or
- [x] There are no security aspects to these changes

### CODE REVIEW

File: pkg/authenticator/k8s/authenticator.go [hunk 1]
```diff
@@ -423,3 +424,17 @@ func consumeInjectClientCertError(path string) string {
 
 	return string(content)
 }
+
+func logIPS(host string) {
+	ips, lookUpError := net.LookupIP(host)
+	if lookUpError != nil {
+		fmt.Fprintf(os.Stderr, "Could not get IPs: %v\n", lookUpError)
+		os.Exit(1)
```
Comment (tzheleznyak) [inline pkg/authenticator/k8s/authenticator.go:432]: Dont we want to return error if error happens and dont exit here like other places in this file?

File: pkg/authenticator/k8s/authenticator.go [hunk 2]
```diff
@@ -423,3 +424,17 @@ func consumeInjectClientCertError(path string) string {
...
+	}
+	ipsList := " IP Addresses: "
```
Comment (tzheleznyak) [inline pkg/authenticator/k8s/authenticator.go:434]: There is a join function of any other allegant string function that can do what you doing here

File: pkg/authenticator/k8s/authenticator.go [hunk 3]
```diff
@@ -423,3 +424,17 @@ func consumeInjectClientCertError(path string) string {
...
+	for _, ip := range ips {
+		ipsList += ip.String()
```
Comment (tzheleznyak) [inline pkg/authenticator/k8s/authenticator.go:436]: If you remain in this way maybe this two lines can be combined to one with string formatting

**Review by diverdane — APPROVED** [review]
LGTM!!!

**Pull request merged**